pandas/core/generic.py

```diff
@@ -3521,6 +3519,10 @@ def interpolate(self, method='linear', axis=0, limit=None, inplace=False,
         dtype: float64
 
         """
+
+    @Appender(_shared_docs['interpolate'] % _shared_doc_kwargs)
+    def interpolate(self, method='linear', axis=0, limit=None, inplace=False,
```

Comment:
Where exactly? In the `Resampler` class documentation or in the `interpolate` method?